apps/prairielearn/assets/scripts/instructorFileEditorClient.ts

```diff
@@ -117,6 +155,36 @@ class InstructorFileEditor {
     this.checkDiff();
     this.editor.resize();
   }
+
+  reformatJSONFile() {
+    const contents = this.editor.getValue();
+    const cursorOffset = getCursorOffsetFromCursorPosition(
```

Comment:
Nit: you have two variables named `cursorOffset` in this function. Maybe rename one of them to avoid confusion?

Reply:
Done in 82b7e00.